roles/hammer_credentials/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Install hammer
+  package:
+    name: tfm-rubygem-hammer_cli_katello
```

Comment:
I was under the impression we had some good provides to make it easier (like we have `foreman-plugin-$plugin` instead of `tfm-rubygem-foreman_$plugin`) but looks like we don't. Perhaps good to introduce now, even though we can't rely on it yet.